airbyte-cdk/python/airbyte_cdk/sources/declarative/extractors/dpath_extractor.py

```diff
@@ -64,7 +63,7 @@ def __post_init__(self, parameters: Mapping[str, Any]) -> None:
             if isinstance(self.field_path[path_index], str):
                 self._field_path[path_index] = InterpolatedString.create(self.field_path[path_index], parameters=parameters)
 
-    def extract_records(self, response: requests.Response) -> Iterable[Mapping[str, Any]]:
+    def extract_records(self, response: requests.Response) -> Iterable[MutableMapping[Any, Any]]:
```

Comment:
Shouldn't the keys always be `str` here?

Reply:
🤦‍♂️ yes